Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.7-alpine
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
```

Comment:
Why did you set these env variables? İs it just c/p from an existing project or did you have any problems?

Reply:
Mostly practice set by other projects I guess. Let me know if you want me to remove them.

Reply:
I went ahead and removed them as not having them didn't produce issues.